app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -161,6 +162,46 @@ const PageRequestReview = ({
               </ChangeApprovalOperations>
             )}
 
+            <h5>Recommended actions before launch</h5>
+            <Table bordered data-testid="table-signoff" className="mb-4">
+              <tr data-testid="table-signoff-qa">
+                <td>
+                  <strong>QA Sign - Off</strong>
```

Comment:
Drive-by feedback: can we pretty please make "Sign - Off" one word, "signoff", or make it a hyphenated word, "sign-off"? The way it currently is makes it seem like an en-dash

Reply:
@jaredlockhart 😢